ChangeLog.txt

```diff
@@ -2,6 +2,7 @@
 * Added Batch cmdlets
     * Start-AzureBatchPoolResize
     * Stop-AzureBatchPoolResize
+* Added support for scaling in Set-AzureRedisCache, using redisConfiguration instead of maxMemoryPolicy, changes related to ResourceBase, ResourceBaseExtended and ResourceIdentity #513
```

Comment:
Normally we do this like this:

* RedisCache cmdlets
    * Set-AzureRedisCache - added support for ...